src/examples/algorithms/HelloSP_TP.cpp

```diff
@@ -1,96 +0,0 @@
-/* ---------------------------------------------------------------------
```

Comment:
I don't know if the Hello example is very good or not but it should be using very explicit inputs to illustrate how things work. The hotgym data wouldn't be good for this. Please revert this and if you still have strong feelings we can discuss elsewhere since it doesn't have anythign to do with this PR.